QUICKSTART-ECS.md

```diff
@@ -0,0 +1,193 @@
+
+You'll also need [aws-cli](https://aws.amazon.com/cli/) set up to interact with AWS.
+
+
+## Create a cluster
+
+An Amazon ECS cluster is a logical grouping of tasks, services, and container instances.
+For more information about clusters, see
+[Amazon ECS clusters](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/clusters.html).
+
+You can create a cluster with the AWS CLI as follows:
+
+```
+aws ecs --region us-west-2 create-cluster --cluster-name bottlerocket
+```
+
+> Note: The command above and subsequent examples include the AWS region, so change it from `us-west-2` if you operate in another region.
+
+## Finding an AMI
+
+You can either use an official AMI provided by Amazon or build an AMI yourself using
+[our guide](https://github.com/bottlerocket-os/bottlerocket/blob/develop/BUILDING.md).
+
+The official AMI IDs are stored in
+[public SSM parameters](https://docs.aws.amazon.com/systems-manager/latest/userguide/parameter-store-public-parameters.html).
+To find the `aws-ecs-1` variant for the `x86_64` architecture in the `us-west-2` region, you can use the following command:
+
+```
+aws ssm get-parameter --region us-west-2 --name "/aws/service/bottlerocket/aws-ecs-1/x86_64/latest/image_id" --query Parameter.Value --output text
+```
+
+If you have `jq` and would like a bit more information, try this:
+
+```
+aws ssm get-parameters --region us-west-2 \
+   --names "/aws/service/bottlerocket/aws-ecs-1/x86_64/latest/image_id" \
+           "/aws/service/bottlerocket/aws-ecs-1/x86_64/latest/image_version" \
+   --output json | jq -r '.Parameters | .[] | "\(.Name): \(.Value) (updated \(.LastModifiedDate | gmtime | strftime("%c")) UTC)"'
+```
+
+You can replace the architecture (`x86_64`) and region (`us-west-2`) to look for other images.
+If you know a specific Bottlerocket version you'd like to use, you can replace `latest` with that version.
+
+You can also see all available parameters
+[through the web console](https://us-west-2.console.aws.amazon.com/systems-manager/parameters/#list_parameter_filters=Path:Recursive:%2Faws%2Fservice%2Fbottlerocket).
+
+## Launching your first instance
+
+In order to launch a Bottlerocket instance into your ECS cluster, you'll first need some information about the resources in your AWS account.
+
+### Subnet info
+
+You should either have a default virtual private cloud (VPC) or have already
+[created a VPC](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/get-set-up-for-amazon-ecs.html#create-a-vpc)
+in your account.
+
+To find your default VPC, run this command.
+(If you use an AWS region other than "us-west-2", make sure to change that.)
+
+```
+aws ec2 describe-vpcs \
+   --region us-west-2 \
+   --filters=Name=isDefault,Values=true \
+   | jq --raw-output .Vpcs[].VpcId
```

Comment:
`.Vpcs[].VpcId` should be in quotes so the shell doesn't interpret it and give an error. Same with line 93

`'.Vpcs[].VpcId'`

Reply:
What shell are you using that you're receiving errors with that?  The command as-written works fine for me with GNU bash 4.4.20.

Reply:
zsh can fail in cases like this.

Reply:
Done